rwgame/GameWindow.cpp

```diff
@@ -31,11 +31,39 @@ void GameWindow::create(const std::string& title, size_t w, size_t h,
         throw std::runtime_error("SDL_GL_CreateContext failed: " + sdlErrorStr);
     }
 
+#include "WindowIcon.h"
+
+    Uint32 rmask, gmask, bmask, amask;
+
+#if SDL_BYTEORDER == SDL_BIG_ENDIAN
+    // Big Endian
+    int offset = (windowIconBitsPerPixel == 32) ? 0 : 8;
```

Comment:
In this case the image buffer is always 32-bit, since nobody really uses square icons nowadays. I'll just remove the bpp lines altogether, the image's hardcoded anyway.